### DISCUSSION TITLE
How to change the an apple Id linked to a fastlane

### DISCUSSION THREAD

**Discussion by dhaivat28** [fastlane/fastlane#17497] [Q&A]
we are currently using fastlane + circle CI to deploy the app to testFlght. But we recently thought of using a common shared apple id rather then using anyone personal account. I am looking forward to creating an apple id for ci machine and replacing it totally in the fastlane. I know we can nuke the certificates and provisioning profile from the repository + app store connect.  But, I am fairly new to the fastlane and would like if anyone could please breif the steps to follow to replace fastlane with a new apple id  and avoid any disaster. Many thanks

**Comment by max-ott** [comment]
Hi there, not really an option any more to create a new Apple ID as Apple requires you to enable 2FA/2Step for any new Apple ID. Maybe you have an existing / old shared Apple ID you can use. Simply change the login variables.

**Comment by wajahatlateef** [comment]
Hi, Sorry for the late reply.
We can change the apple-id linked to Fastlane for uploading of an app to Appstore. please follow the below steps

1. Search for deliver. In keychain on mac and delete the entry for older apple-id linked with fastlane.
2. Create FASTLANE_SESSION  by executing the bellow command
      fastlane spaceauth -u <new-appleid> 
      export the output displayed as env var
4.  Create FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD on http://appleid.apple.com/
